examples/resize/termsize.go

```diff
@@ -0,0 +1,39 @@
+// https://stackoverflow.com/a/15784352/587091
+
+package main
+
+import (
+	"runtime"
+	"syscall"
+	"unsafe"
+)
+
+const (
+	TIOCGWINSZ     = 0x5413
+	TIOCGWINSZ_OSX = 1074295912
```

Comment:
What does this mean? Try and make these constants more clear and prefer Go-style CamelCase (public) or snakeCase (private) identifiers. C-style constant naming is reserved for imported and exposed C constants.

More explanation if you're interested: https://stackoverflow.com/a/22688926

Reply:
I was able to find a constant, `syscall.TIOCGWINSZ`, that I _think_ will vary portably, which reduces complexity here a lot. [`syscall` reference](https://pkg.go.dev/syscall) shows `TIOCGWINSZ` as `0x5413`, and [`darwin` variant reference](https://pkg.go.dev/syscall?GOOS=darwin) shows `TIOCGWINSZ` as  `0x40087468`, both of which match what I had.

As for the name itself, it's covered [here](https://stackoverflow.com/a/16237327/587091):
>Thus the name `TIOCGWINSZ`, `IOC`tl to `G`et the `WIN`dow `S`i`Z`e.